src/components/footer/footer.tsx

```diff
@@ -3,32 +3,34 @@ import { Link } from '@builder.io/qwik-city';
 import { APP_STATE } from '~/constants';
 import GitIcon from '../icons/GitIcon';
 
-export const navigation = {
-	support: [
-		{ name: 'Help', href: '#' },
-		{ name: 'Track order', href: '#' },
-		{ name: 'Shipping', href: '#' },
-		{ name: 'Returns', href: '#' },
-	],
-	company: [
-		{ name: 'About', href: '#' },
-		{ name: 'Blog', href: '#' },
-		{ name: 'Corporate responsibility', href: '#' },
-		{ name: 'Press', href: '#' },
-	],
-};
 export default component$(() => {
 	const collections = useContext(APP_STATE).collections.filter(
 		(item) => item.parent?.name === '__root_collection__' && !!item.featuredAsset
 	);
+
```

Comment:
It needs to move the `navigation` object to the component context in order to get the locale info properly.